src/components/SettingsDialog/Participants/Participants.tsx

```diff
@@ -95,28 +117,35 @@ export const Participants = () => {
             .filter((participant) => participant.connected === onlineFilter)
             .map((participant) => (
               <li key={participant.user.id} className="participants__list-item">
-                <UserAvatar avatar={participant.user.avatar} className="participant__avatar" id={participant.user.id} ready={participant.ready} title={participant.user.name} />
-                <div className="participant__name-role-wrapper">
-                  <span className="participant__name">{participant.user.name}</span>
-                  {participant.role === "OWNER" || !isModerator || participant.user.id === self.user.id ? (
+                <UserAvatar
+                  avatar={participant.user.avatar}
+                  className={classNames("participant__avatar", {banned: participant.banned})}
+                  id={participant.user.id}
+                  ready={participant.ready}
+                  title={participant.user.name}
+                />
+                <div className={classNames("participant__name-role-wrapper", {banned: participant.banned})}>
```

Comment:
Instead of adding a banned class to the avatar and the name role wrapper, wouldn't it be more conveniant to add the banned class to the parent? 

```css
.participants__list-item--banned > .participant__avatar {...}
.participants__list-item--baned > .participant__name-role-wrapper {...}
```

Reply:
I think this is a personal preference.
I had a similar thought, but didn't do it because it was too bloaty in my eyes.
However, I'll change it to the following:

```css
  &.banned > *:not(.participant__join-icon) {
    opacity: 0.6;
  }
```

Let me know what you think about it :)